execution/executor-benchmark/src/main.rs

```diff
@@ -96,6 +96,8 @@ pub struct PipelineOpt {
     num_executor_shards: usize,
     #[clap(long)]
     async_partitioning: bool,
+    #[clap(long)]
+    use_global_executor: bool,
```

Comment:
(the same idea as above) This is very specific to V1/V2 and should be inside some "MultiRoundPartitionerConfig" rather than a direct member of `PipelineConfig`.

Reply:
Good point. How about I take this as a follow up to clean up/reorganize some configs related to sharding in executor benchmark?